MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -200,11 +200,25 @@ await SaveImage(
                 // TODO: Isolate this hack into the tvh plugin
                 if (string.IsNullOrEmpty(contentType))
                 {
+                    // First, check for imagecache special case
                     if (url.Contains("/imagecache/", StringComparison.OrdinalIgnoreCase))
                     {
-                        contentType = MediaTypeNames.Image.Png;
+                        contentType = "image/png";
```

Comment:
Why change from the .net provided constant?

Reply:
Since .NET doesn't offer a constant for the webp MIME type, I chose to use raw strings for consistency. I'll go ahead and replace them with the .NET constants.